Vienna/Sources/Main window/BrowserTab.swift

```diff
@@ -33,19 +33,18 @@ class BrowserTab: NSViewController {
 		prefs.plugInsEnabled = true
 		let config = WKWebViewConfiguration()
 		config.preferences = prefs
-		if #available(OSX 10.11, *) {
-			config.applicationNameForUserAgent = "vienna-rss"
-			config.allowsAirPlayForMediaPlayback = true
-		}
+		// for useragent, we mimic the installed version of Safari and add our own identifier 
+		let shortSafariVersion = Bundle(path: "/Applications/Safari.app")?.infoDictionary?["CFBundleShortVersionString"] as? String
+		let viennaVersion = (NSApp as? ViennaApp)?.applicationVersion?.prefix(while: { character in return character != " " })
+		config.applicationNameForUserAgent = "Version/\(shortSafariVersion ?? "9.1") Safari/605 Vienna/\(viennaVersion ?? "3.5+")"
+		config.allowsAirPlayForMediaPlayback = true
```

Comment:
I think those two lines should be kept inside the #if available branch, for consistency and backwards compatibility just in case. We can later remove those in a separate commit.